defender/tests.py

```diff
@@ -75,7 +75,7 @@ def _login(
     def setUp(self):
         """ Create a valid user for login
         """
-        self.user = User.objects.create_superuser(
+        self.user = get_user_model().objects.create_superuser(
```

Comment:
Does it related to the fix?

Reply:
it doesn't but it's idiomatic django for applications that use custom user models. I added it to get past that exception in my local tests. I can remove it if you'd like?

Reply:
I just reverted this change